module-2/controllers/user.js

```diff
@@ -0,0 +1,99 @@
+const userRouter = require('express').Router()
+const uuid = require('uuid')
+const Joi = require('joi')
+const { validateSchema } = require('../utils')
+
+let users = []
+
+const getAutoSuggestUsers = (loginSubstring, limit) => {
+  return users
+    .filter((item) => item.login.includes(loginSubstring))
+    .filter((item, i) => i <= limit - 1)
```

Comment:
you can use splice here to limit the users

Reply:
splice will modify the original array, hence cannot be used in this scenario